src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -557,7 +561,9 @@ def _parse_requests(self, requests: dict) -> LLMMetrics:
             inter_token_latencies.append(itl_per_request)
 
         # request & output token throughput
-        benchmark_duration = (max_res_timestamp - min_req_timestamp) / 1e9  # nanosec
+        benchmark_duration = scale(
```

Comment:
Since we're mixing unit types, my preference would be appending the unit e.g. `benchmark_duration_s`. That's clearer to me than comments and consistent with how it's done elsewhere in our code. It avoids the need to accurately know the unit for each metric.